.circleci/config.yml

```
@@ -5,9 +5,8 @@ jobs:
   python:
     docker:
       - image: udata/circleci:2-alpine
-      - image: mongo:3.6
+      - image: mongo:5.0.15
```

We can already upgrade to
```suggestion
      - image: mongo:6.0.4
```